go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/coredns/coredns
 
-go 1.20
+go 1.21
+
+toolchain go1.21.8
```

Comment:
Let's avoid adding the `toolchain` for now.

```suggestion
```

Reply:
Updated